js/controller.js

```diff
@@ -259,6 +269,19 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
       this.accessibilityControls = new AccessibilityControls(this); //keyboard support
       this.state.screenToShow = CONSTANTS.SCREEN.INITIAL_SCREEN;
+
+      if (this.getVrParams && this.getVrParams()) {
+        this.state.isVideo360 = true;
+        this.mb.subscribe(OO.EVENTS.DIRECTION_CHANGED, 'customerUi', _.bind(this.setViewingDirection, this));
+      }
+    },
+
+    getVrParams: function(){
+      var playerParam = this.state.playerParam;
+      var bitWrapper = playerParam ? playerParam['bit-wrapper'] : null;
+      var isVr = !!bitWrapper && !!bitWrapper.source && !!bitWrapper.source.vr;
+
+      return isVr ? _.extend({}, bitWrapper.source.vr) : false;
```

Comment:
Please do not call functions that change values of variables inside of a turnary statement. Stick this inside of an if statement. Otherwise using the debugger to debug this will be much harder.